experimental/segmenter/src/lstm_bies.rs

```diff
@@ -102,8 +124,8 @@ impl<'l> Lstm<'l> {
 
     // TODO(#421): Use common BIES normalizer code
     /// `compute_bies` uses the computed probabilities of BIES and pick the letter with the largest probability
-    fn compute_bies(&self, arr: Array1<f32>) -> Result<char, Error> {
-        let ind = math_helper::max_arr1(arr.view());
+    fn compute_bies(&self, arr: MatrixBorrowed<1>) -> Result<char, Error> {
```

Comment:
preexisting, but maybe fix: this function is called in a single place which uses `unwrap_or('s')`, so there's unnecessary logic in here

Reply:
Yep I can clean this up